tests/test_generators.py

```diff
@@ -7,11 +7,11 @@
 from .util import run_in_separate_process, xfail_on_ci_if_model_is_incompatible
 
 # generate list of all models in impy.models
-models = set(obj for obj in im.__dict__.values() if type(obj) is abc.ABCMeta)
+models = list(obj for obj in im.__dict__.values() if type(obj) is abc.ABCMeta)
```

Comment:
Why do you change this?

Reply:
If you want a list then just use [...] not list(...). However, that is also true for the original, set(...) can be replaced with {...} when ... is a generator expression. Doing this generates slightly faster bytecode in Python and it more pythonic.

Reply:
List is needed for launching tests parallel using

```python
pip install pytest-xdist[psutil]
pytest -vv -n auto  # <- This code runs the tests in parallel.
```
The problem is that it doesn't work with set(...).

I will change list(...) to [...]

Reply:
Ok, I changed all the files, since we use this everywhere, I put the code into a function.